src/twisted/internet/test/test_inlinecb.py

```diff
@@ -145,3 +145,35 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         self.assertIn("in erroring", f.getTraceback())
+
+    def test_forwardLotsOfTracebacks(self):
+        """
+        Several Chained inlineCallbacks gives information about all generators
+
+        Reproducing #9176
```

Comment:
the docstring for a test should not make a reference to any bugfix, but rather be self-contained and fully describe the performed action and the expected result.

Please use full sentences (with a full stop at the end) and with proper cases.

The full stop is of great help to hint whether the information is complete, or if the sentence was accidentally left unfinished.